recon/src/ahash.rs

```diff
@@ -1,17 +1,18 @@
 use ::serde::{de::Error, Deserialize, Deserializer, Serialize, Serializer};
 use multihash::{Hasher, Sha2_256};
+use std::convert::From;
 use std::fmt::{self, Debug, Display, Formatter};
 
-use crate::Hash;
+use crate::AssociativeHash;
 
 /// AHash an associative hash function for use in set reconciliation
 #[derive(Default, PartialEq, Clone, Copy, Deserialize)]
-pub struct AHash(#[serde(deserialize_with = "deserialize_from_bytes")] [u32; 8]);
+pub struct ShaSum256(#[serde(deserialize_with = "deserialize_from_bytes")] [u32; 8]);
```

Comment:
Would it make more sense to call it *SumSha256* or *Sha256Sum*? That way we don't break up "sha256"